addons/website/static/src/js/menu/seo.js

```diff
@@ -405,17 +405,18 @@ var SeoConfigurator = Dialog.extend({
         ['/website/static/src/xml/website.seo.xml']
     ),
     events: {
-        'keyup input[name=seo_page_keywords]': 'confirmKeyword',
-        'blur input[name=seo_page_title]': 'titleChanged',
-        'blur textarea[name=seo_page_description]': 'descriptionChanged',
+        'input input[name=seo_page_keywords]': 'confirmKeyword',
```

Comment:
This is too laggy when typing fast in the input and textarea, but it is cool to see changes in live.
Can you try with _.throttle ?
(Same for every input event on seo.js)

Reply:
I tried but it feels laggy now because of the delay. What was the lag problem before, maybe only because of the debugger open?

Reply:
Please try again when the currently running runbot is ready.

Reply:
issue is gone, adding self.$selector fixed the problem